docs/source/use_cases/privacy_protection_faces.md

```diff
@@ -0,0 +1,61 @@
+# Privacy Protection (Faces)
+
+## Overview
+
+As organisations collect more data, there is a need to better protect the identities of individuals in public and private places. AI Singapore has developed a solution that performs face anonymisation. This can be used to comply with the General Data Protection Regulation (GDPR) or other data privacy laws.
+
+Our solution automatically detects and mosaics (or blurs) human faces. This is further elaborated in a [subsequent section](#how-it-works).
+
+## Demo
+
+To try our solution on your own computer with [PeekingDuck installed](../getting_started/01_installation.md): use the following configuration file: [privacy_protection_faces.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/use_cases/privacy_protection_faces.yml) and run PeekingDuck.
+
+```
+> peekingduck run --config_path <path_to_privacy_protection_faces.yml>
+```
+
+## How it Works
+
+There are two main components to face anonymisation: 1) face detection using AI; and 2) face de-identification. 
+
+**1. Face Detection**
+
+We use an open source face detection model known as [MTCNN](https://arxiv.org/abs/1604.02878) to identify human faces. This allows the application to identify the location of human faces in a video feed. The location is returned as two (x, y) coordinates in the form [x1, y1, x2, y2], where (x1, y1) is the top-left corner of the bounding box, and (x2, y2) is the bottom-right. These are used to form the bounding box of each human face detected. For more information on how adjust the MTCNN node, checkout the [MTCNN configurable parameters](/peekingduck.pipeline.nodes.model.mtcnn.Node).
+
+We can also use the [Yolo face model](/peekingduck.pipeline.nodes.model.yolo_face.Node) as an alternative. The yolo face model is a two-class model capable of differentiating human faces with and without face masks.
+
+**2. Face De-Identification**
+
+To perform face de-identification, we pixelate or gaussian blur the areas bounded by the bounding boxes.
+
+## Nodes Used
+
+These are the nodes used in the earlier demo (also in [privacy_protection_faces.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/use_cases/privacy_protection_faces.yml)):
+```
+nodes:
+- input.live
+- model.mtcnn
+- dabble.fps
+- draw.mosaic_bbox
+- draw.legend
+- output.screen
+```
+
+**1. Face Detection Node**
+
+As mentioned, we use the MTCNN model for face detection. You can also try the [Yolo face model](/peekingduck.pipeline.nodes.model.yolo_face.Node) that is included in our repo. By default, it uses the Yolov4-tiny model to detect faces. For better accuracy, you can change the parameters in the run config declaration to use the Yolov4 model instead.
```

Comment:
Should we mention that if MTCNN has trouble with masked faces, can try yolo instead?